### PR TITLE
Version changes based on stable20160913 updates

### PR DESCRIPTION
**Pull request by kbrownell**
This pull request includes version changes to contrib and stanford modules that were updated on existing sites as part of the 2016-09-13 through 09-15 updates.  Notably: stanford_courses went from 3.x to 4.x and we moved the stanford hosted version of nobots to Drupal's new project.

### CODE REVIEW

File: development/product/jumpstart/stanford.make [hunk 1]
```diff
@@ -70,11 +88,11 @@ projects[stanford_image_styles][download][type] = "git"
 projects[stanford_image_styles][download][url] = "git@github.com:SU-SWS/stanford_image_styles.git"
 projects[stanford_image_styles][download][branch] = "7.x-3.x-dev"
 
-projects[nobots][subdir] = "stanford"
-projects[nobots][type] = "module"
-projects[nobots][download][type] = "git"
-projects[nobots][download][url] = "git@github.com:SU-SWS/nobots.git"
-projects[nobots][download][branch] = "7.x-1.x"
+projects[stanford_news][type] = "module"
```
Comment (sherakama) [inline development/product/jumpstart/stanford.make:91]: Please confirm. This is because news and publication are going in to the sites "stack"?

Comment (jbickar) [reply development/product/jumpstart/stanford.make:91]: Confirmed, and SWB-approved.

Comment (sherakama) [reply development/product/jumpstart/stanford.make:91]: 👍  Thank you kindly.

**Comment by jbickar** [comment]
Starting dev builds of each.

`grunt build:make:install --build-git-branch="stable20160913"`

**Comment by jbickar** [comment]
dev builds all complete and install for:
- jsv
- jsplus
- jsa
- jsvpsa
- jse

Did not test JSL or Drupalcamp

**Comment by sherakama** [comment]
JSL is in development and DrupalCamp... well... It's camp.
I am good with that list above.

**Comment by jbickar** [comment]
JSVPSA prod build fails for very good reasons.

**Comment by jbickar** [comment]
Prod builds all complete and install for:
- jsv
- jsplus
- jsa
- jse

**Comment by jbickar** [comment]
JSV prod Behat passes

**Comment by jbickar** [comment]
JS+ prod Behat:

```
71 scenarios (71 passed)
357 steps (357 passed)
9m10.01s (19.88Mb)
```

**Comment by jbickar** [comment]
JSA prod Behat:

```
83 scenarios (83 passed)
303 steps (303 passed)
6m52.03s (16.90Mb)
```

**Comment by jbickar** [comment]
JSE prod Behat failures (running `behat -p local -s all features/`):

```
    features/jse_courses_importer.feature:7
    features/jse_event_series.feature:31
    features/jse_full_html.feature:23
    features/jse_get_help.feature:7
    features/jse_help.feature:19
    features/jse_metatag.feature:7
```

In re-running those:
- `jse_courses_importer.feature` fails because `stanford_courses` is not enabled
- `jse_full_html.feature` fails because it's looking for "Stanford Page Foo has been saved" and the text is "Stanford Page Foo has been updated"
- `jse_metatag.feature` fails because "Stanford Page" is not an option for "Type" at `admin/config/search/metatags/config/add`. I have no idea why. `stanford_page` is enabled.

The other failures are known issues, I believe.

I'll let @boznik look at the issues above.

**Comment by jbickar** [comment]
Hm, looks like `cube`, `rubik` and `tao` are going into `sites/all/modules`

**Comment by jbickar** [comment]
JSVPSA dev Behat, only 1 failure:
`includes/features/SU-SWS/stanford_news/stanford_news.feature:48`

**Comment by jbickar** [comment]
Next to @sherakama. I have built and tested all products as appropriate and as detailed above. Please review the changes for sanity and then pass to @boznik.

**Review by sherakama — COMMENTED** [review]
Looking pretty great. Just a couple of minor comments.

Why was stanford_framework rolled back from 3.2-dev+8 to 3.2? 

Why was rubik rolled back to 4.2?

**Comment by jbickar** [comment]
`stanford_framework` was probably an oversight. I can push a fix. Should it be `3.2-dev+8` for all products?

`rubik` is on 4.2 on the stack and we're not using it, so I'd prefer not to have a one-off.

**Comment by sherakama** [comment]
Oh I just looked at the releases on stanford_framework. They are not following the same conventions as we do. 3.2 IS the latest. Go with 3.2.

**Comment by boznik** [comment]
jsv dev build:
--- Failed scenarios:

```
/Users/ggarvey/www/linky_clicky/includes/features/all_sites.feature:37
features/stanford_jumpstart/stanford_jumpstart_wysiwyg/stanford_jumpstart_wysiwyg.feature:26
```

70 scenarios (68 passed, 2 failed)
317 steps (315 passed, 2 failed)
21m18.43s (15.95Mb)

**Comment by boznik** [comment]
jsplus dev build:
--- Failed scenarios:

```
/Users/ggarvey/www/linky_clicky/includes/features/all_sites.feature:37
features/editor.feature:37
features/stanford_jumpstart/stanford_jumpstart_wysiwyg/stanford_jumpstart_wysiwyg.feature:26
```

71 scenarios (68 passed, 3 failed)
357 steps (354 passed, 3 failed)
13m30.22s (17.47Mb)

**Comment by boznik** [comment]
jsa dev build:
--- Failed scenarios:

```
/Users/ggarvey/www/linky_clicky/includes/features/all_sites.feature:37
features/jsa_site_owner.feature:45
features/site_owner.feature:45
```

83 scenarios (80 passed, 3 failed)
303 steps (300 passed, 3 failed)
9m22.26s (15.42Mb)

**Comment by boznik** [comment]
This looks good to me for jse.

**Comment by jbickar** [comment]
w00t.

**Pull request merged**